lua/rzls/handlers/providehtmldocumentcolor.lua

```diff
@@ -0,0 +1,39 @@
+local documentstore = require("rzls.documentstore")
+local razor = require("rzls.razor")
+local Log = require("rzls.log")
+
+local empty_response = {}
+
+---@param _err lsp.ResponseError
+---@param result lsp.DocumentDiagnosticParams
+---@param _ctx lsp.HandlerContext
+---@param _config table
+return function(_err, result, _ctx, _config)
+    assert(not _err, vim.inspect(_err))
```

Comment:
```suggestion
return function(err, result, _ctx, _config)
    assert(not err, vim.inspect(err))
```
only unused params are prefixed with a `_`